src/core/execution/NukeExecution.ts

```diff
@@ -221,6 +223,10 @@ export class NukeExecution implements Execution {
         }
       }
     }
+
+    this.redrawBuildings(
+      (magnitude.outer + SPRITE_RADIUS) * (magnitude.outer + SPRITE_RADIUS),
+    );
```

Comment:
How do you feel about this (and the corresponding change below)?

```suggestion
    this.redrawBuildings(magnitude.outer + SPRITE_RADIUS);
```

Reply:
Done ✅